.github/workflows/maven.yaml

```diff
@@ -9,6 +9,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

this fetches all refs, including origin/master